client/extensions/woocommerce/components/custom-tree-selector/README.md

```diff
@@ -0,0 +1,61 @@
+Custom tree selection component. Renders checkboxes for each node in the tree.
```

Comment:
This file needs a header. It could also use an example of the component itself, I skimmed through the readme but missed that the prop `nodes` when trying it out. Maybe check out the format of other READMEs, like [OrderSelect](https://github.com/Automattic/wp-calypso/blob/master/client/extensions/woocommerce/components/order-status/README.md) or [Card](https://github.com/Automattic/wp-calypso/blob/master/client/components/card/README.md).

Reply:
Thanks for the feedback. I'll look those over and revise. 👍